### PR TITLE
adds info text for aggregate report form. adds checkbox group styles …

### PR DESCRIPTION
**Pull request by esotrope**
…intended for other package

### CODE REVIEW

File: webapp/src/main/webapp/src/styles.less [hunk 1]
```diff
@@ -331,6 +331,36 @@ body .select-menu-group {
   padding: 0.5em 0.5em 0.5em 0;
 }
 
+/**
```
Comment (esotrope) [inline webapp/src/main/webapp/src/styles.less]: these styles were supposed to be wrapped in the checkbox-none-state PR but i committed them to the wrong branch so i'm including them here

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 2]
```diff
@@ -608,36 +608,36 @@
...
-            "info": "Limited English Proficiency"
+            "info": "Students who use a language other than English as their primary language and who have been assessed as not possessing the English language skills necessary to succeed in the regular instructional program."
           },
           "economic-disadvantage": {
             "label": "Economic Disadvantage",
-            "info": "Economic Disadvantage"
+            "info": "Indicates that a student qualifies for free or reduced lunch or meets other state criteria for economic disadvantage."
           },
           "achievement-level-display-type": {
             "label": "Achievement Level",
```
Comment (esotrope) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: @jtreuting The following form items do not have popover info text yet.
I'm thinking they don't all need it but let me know and I'll write something.

```
Assessment Grades
Assessment Type
Academic Years
Subjects
Achievement Level (separate or grouped)
Show Value as (percent or number)
Comparative Subgroups
```

Comment (jtreuting) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: Don't write anything yet.  Once we have a stable UI on this page we will ask SB to provide info.  Though some of these might have context info from other screens.  If there is and they make sense we should reuse them for consistency.

Comment (esotrope) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: Alright - will do

**Pull request merged**